tools/shared.py

```diff
@@ -251,13 +251,8 @@ def run_js_tool(filename, jsargs=[], node_args=[], **kw):  # noqa: mutable defau
 
 
 def get_npm_cmd(name):
-  if WINDOWS:
-    cmd = [path_from_root('node_modules/.bin', name + '.cmd')]
-  else:
-    cmd = config.NODE_JS + [path_from_root('node_modules/.bin', name)]
-  if not os.path.exists(cmd[-1]):
-    exit_with_error(f'{name} was not found! Please run "npm install" in Emscripten root directory to set up npm dependencies')
```

Comment:
We have the `bootstrap` checker that makes this error message very unlikely so I don't think its worth preserving.